docs/source/scheduler-overview.rst

```diff
@@ -0,0 +1,146 @@
+Scheduler Overview
+==================
+
+Once a dask graph is created, a scheduler is needed to run it. Dask currently
```

Comment:
Consider active voice:

    After we create a dask graph we use a scheduler to run it.

Or